inference-engine/tests/functional/inference_engine/transformations/convert_scatter_elements_to_scatter_test.cpp

```diff
@@ -33,7 +33,7 @@ std::shared_ptr<ngraph::Function> get_initial_function(const ngraph::PartialShap
     auto updates = std::make_shared<ngraph::opset3::Parameter>(ngraph::element::f32, updates_shape);
     auto axis_const = ngraph::opset3::Constant::create(ngraph::element::i64, {1}, {axis});
 
-    uint64_t broadcast_len = broadcast_shape.rank().get_length();
+    size_t broadcast_len = broadcast_shape.rank().get_length();
```

Comment:
I suggest not to use `size_t` but `int64_t` like here: https://github.com/openvinotoolkit/openvino/pull/1721/

Otherwise we will have invalid numbers on 32bit platform

Reply:
> Otherwise we will have invalid numbers on 32bit platform

How? `broadcast_len` is later used in `vector<size_t>`

Reply:
https://github.com/openvinotoolkit/openvino/blob/3c9fc72b5888bbeca005746baf11815defbb8ce0/ngraph/core/include/ngraph/dimension.hpp#L34-L37
and
https://github.com/openvinotoolkit/openvino/blob/3c9fc72b5888bbeca005746baf11815defbb8ce0/ngraph/core/include/ngraph/dimension.hpp#L68

Reply:
I agree with @dkurt , sometime ago we had a lot of issues with `size_t`

Reply:
@dkurt @ilyachur what exactly do you want to change? Later this variable is used as a value in `ngraph::Shape` container, which is `vector<size_t>`

We need to change `ngraph::Shape` to be `vector<int64_t>`, otherwise we cannot avoid this conversion, is not it?

Reply:
The problem is that for 32bit OS in case of conversion from `int64_t` to `size_t` with lead to large numbers (see ticket 36799, for example). So it's OK to use `auto` but definitely not `size_t` because on 32bit OS it's 32bit unsigned while `get_length` is 64bit signed integer.

Reply:
But at the current moment we are talking about the `Dimension` class, and `get_lenght()`  method in this class returns `int64_t`.

Reply:
> So it's OK to use auto but definitely not size_t

It's converted in `size_t` anyway in the code below, so it does not matter where to convert

Reply:
Added a check that `broadcast_len` range can be represented in `size_t`